examples/sklearn_elasticnet_wine/train.py

```diff
@@ -34,7 +34,7 @@ def eval_metrics(actual, pred):
 
     # Read the wine-quality csv file from the URL
     csv_url = (
-        "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/data/winequality-red.csv"
```

Comment:
The referenced file was renamed in https://github.com/mlflow/mlflow/pull/8043, but I missed the reference in this example. Using the official website is probably better.